docs/src/operations/requirements.md

```diff
@@ -114,4 +115,4 @@ be limited to any free 13 port range with `--dynamic-port-range`
 For security purposes, it is not suggested that the following ports be open to
 the internet on staked, mainnet-beta validators.
 - 8899 TCP - JSONRPC over HTTP. Change with `--rpc-port RPC_PORT``
-- 8900 TCP - JSONRPC over Websockets. Derived. Uses `RPC_PORT + 1`
\ No newline at end of file
+- 8900 TCP - JSONRPC over Websockets. Derived. Uses `RPC_PORT + 1`
```

Comment:
Cannot figure out why this line is in the diff; different newline character ?

Reply:
Yep, removed newline